src/test/java/org/apache/commons/codec/BinaryEncoderAbstractTest.java

```diff
@@ -34,13 +35,8 @@ public void testEncodeEmpty() throws Exception {
     }
 
     @Test
-    public void testEncodeNull() throws Exception {
+    public void testEncodeNull() {
         final BinaryEncoder encoder = makeEncoder();
-        try {
-            encoder.encode(null);
-            fail("EncoderException exptected");
-        } catch (final EncoderException ee) {
-            // An exception should be thrown
-        }
+        assertThrows(EncoderException.class, () -> { encoder.encode(null); });
```

Comment:
@itamarc 
Thank you for your PR.
You do not need the curly brackets here, less clutter is better.